src/CsWin32Generator/Program.cs

```diff
@@ -106,6 +106,11 @@ public async Task<int> Main(
             AllowMultipleArgumentsPerToken = true,
         };
 
+        var assemblyNameOption = new Option<string?>("--assembly-name")
+        {
+            Description = "The name of the assembly being generated for.",
```

Comment:
This doc should specify that this is the _full_ name of the assembly, since the public key token is a critical part to the InternalsVisibleTo check.

Reply:
I've added a new parameter for the key file.